infra/cmake/modules/ExternalBuildTools.cmake

```diff
@@ -60,6 +60,8 @@ function(ExternalBuild_CMake)
                   RESULT_VARIABLE BUILD_EXITCODE)
```

Comment:
maybe this file is added mistakenly?

Reply:
It seems that the error occurs because of building tests in pybind11.

```
20:31:52   CMake Error at tests/CMakeLists.txt:197 (message):
20:31:52 
20:31:52     Running the tests requires pytest.  Please install it manually (try:
20:31:52     /usr/bin/python3.5 -m pip install pytest)
20:31:52 
20:31:52   
20:31:52 
20:31:52   
20:31:52 
20:31:52   -- Configuring incomplete, errors occurred!
```

I'm trying to turn off the option to build pybind11 tests.

Reply:
I could prevent building pybind11 tests by adding EXTRA_OPTS as below.
```
  nnas_include(ExternalBuildTools)
  ExternalBuild_CMake(CMAKE_DIR   ${Pybind11Source_DIR}
                      BUILD_DIR   ${CMAKE_BINARY_DIR}/externals/PYBIND11/build
                      INSTALL_DIR ${EXT_OVERLAY_DIR}
                      IDENTIFIER  "2.3.0"
                      PKG_NAME    "PYBIND11"
                      EXTRA_OPTS "-DPYBIND11_TEST:BOOL=OFF")
```